static/app/views/onboarding/targetedOnboarding/components/createProjectsFooter.tsx

```diff
@@ -42,31 +42,29 @@ export default function CreateProjectsFooter({
 }: Props) {
   const api = useApi();
   const {teams} = useTeams();
+  const [clientState, setClientState] = useOnboardingState();
```

Comment:
naming is hard, but with hooks, you would generally write this is `[$name, set$Name] = use$Name`
```suggestion
  const [persistedOnboardingState, setPersistedOnboardingState] = usePersistedOnboardingState();
```
It's longer, but your future self will thank you :)